cmd/server.go

```diff
@@ -70,6 +70,9 @@ func initServerCmd(cmd *cobra.Command) {
 	cmd.Flags().Int64Var(&serverConfig.GcTTL, "gc-ttl", defaultServerConfig.GcTTL, "CDC GC safepoint TTL duration, specified in seconds")
 	cmd.Flags().StringVar(&serverConfig.LogFile, "log-file", defaultServerConfig.LogFile, "log file path")
 	cmd.Flags().StringVar(&serverConfig.LogLevel, "log-level", defaultServerConfig.LogLevel, "log level (etc: debug|info|warn|error)")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxSize, "log-file-max-size", defaultServerConfig.LogFileMaxSize, "max log file size in MB")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxDays, "log-file-max-days", defaultServerConfig.LogFileMaxDays, "max log file keep days")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxBackups, "log-file-max-backups", defaultServerConfig.LogFileMaxBackups, "maximum number of old log files to retain")
```

Comment:
I don't think we need to expose these settings in command flags.

It is enough to support these settings in TiCDC's config file, just like TiDB https://github.com/pingcap/tidb/blob/516584aaa47bec5a4dc131be95b413b48500f5fa/config/config.toml.example#L168-L179

Reply:
ok, I have modified it. PTAL